db/migrate/20161025192852_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :email, :unique => true
+      t.string :password_digest
+      t.string :uuid
```

Comment:
I would just use the user ID or email as the unique identifier for the user, rather than using a UUID.